Added few faqs (#6749)
harness/developer-hub: kb/continuous-delivery/continuous-delivery-faqs.md

You can use this expression in any setting that supports expressions in Harness.

> #### Is the "Enable Native Helm steady state for jobs" option in default setting is for the steady state check?

Reviewer: ```suggestion
#### Is the "Enable Native Helm steady state for jobs" option a default setting for the steady state check?
```

This selection is solely for verifying the steady state of a Kubernetes Job deployed via the Helm chart.

> #### Is there a way to get a notification whenever a new pipeline gets created?

Reviewer 2: ```suggestion
#### Is there a way to get notified whenever a new pipeline is created?
```

> #### Is there a way to get the notification whenever the new pipeline gets created?

Reviewer: ```suggestion
#### Is there a way to get a notification whenever a new pipeline gets created?
```
